udata_front/theme/gouvfr/assets/js/components/MultiSelect/MultiSelect.vue

```diff
@@ -650,14 +650,22 @@ export default defineComponent({
       }
     };
 
+    function diff(newValue, oldValue) {
+      if (Array.isArray(newValue) && Array.isArray(oldValue)) {
+        return [
+          ...newValue.filter(item => !oldValue.includes(item)),
+        ];
+      } else {
+        return newValue === oldValue ? [] : [newValue];
```

Comment:
why is oldValue not in the 2nd array?

Reply:
It's unnecessary, with only the new one, the value is toggled between the 2 in single select mode